conf/application.conf

```diff
@@ -50,7 +50,7 @@ auth = {
 # A list of known hosts
 hosts = [
   {
-    host = "https://logging-es:9200"
+    host = "https://elastic1:9200"
```

Comment:
Is it possible to parameterize this too?

Reply:
did a quick parameterization